config/locales/en.yml

```diff
@@ -597,12 +597,17 @@ en:
         advanced_settings: Advanced Settings
         update_and_close: Update and Close
         choose_products_from: 'Choose Products From:'
-        pickup_time_tip: When orders from this OC will be ready for the customer
-        pickup_instructions_tip: These instructions are shown to customers after they complete an order
       exchange_form:
+        pickup_time_tip: When orders from this OC will be ready for the customer
```

Comment:
These two lines belong to the last commit, but rewriting the commits now would be a bit too much hassle.